apps/pie-storybook/stories/testing/pie-avatar.test.stories.ts

```diff
@@ -2,33 +2,44 @@ import { html } from 'lit';
 import { type Meta } from '@storybook/web-components';
 
 import '@justeattakeaway/pie-avatar';
-import { type AvatarProps } from '@justeattakeaway/pie-avatar';
-
-import { createStory } from '../../utilities';
+import { type AvatarProps, tags } from '@justeattakeaway/pie-avatar';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { createStory, type TemplateFunction } from '../../utilities';
 
 type AvatarStoryMeta = Meta<AvatarProps>;
-
-const defaultArgs: AvatarProps = {};
+type OptionalAvatarProps = Partial<AvatarProps>; // giving us a type that can have any of the props in the AvatarProps but can miss any of them (allows us to get {})
 
 const avatarStoryMeta: AvatarStoryMeta = {
     title: 'Avatar',
     component: 'pie-avatar',
-    argTypes: {},
-    args: defaultArgs,
-    parameters: {
-        design: {
-            type: 'figma',
-            url: '',
+    argTypes: {
+        label: {
+            description: 'Text for user name',
+            control: {
+                type: 'text',
+            },
+        },
```

Comment:
if you agree with the suggestion [here](https://github.com/justeattakeaway/pie/pull/2465/files#r2270315366) - please update the description here as well